_posts/2021-11-10-release-0.9.13.md

```diff
@@ -0,0 +1,147 @@
+- [__New and improved vehicles__](#new-and-improved-vehicles): A brand new vehicle and opening doors with improved breaking and gear behaviour
+- [__Runtime texture updating__](#runtime-texture-updates): Runtime texture updating through the CARLA Python API
+- [__Python library upgrades__](#python-library-upgrades): carla-agents library now available through PyPi and new Python 3.9 wheels
+
+## Traffic Manager
+
+### User defined paths
+
+The traffic manager has been extended with some great new functionality to extend user control over the behaviour of vehicles in the traffic manager. The API now exposes new traffic manager functions that can be used to guide vehicles with a user-defined custom path.
+
+The `set_path` function allows a custom route to be defined using coordinates defined as CARLA Locations:
+
+```py
+path = [carla.Location(x=-506.696198, y=179.384308, z=0.038194),
+                    carla.Location(x=-504.745972, y=232.868927, z=0.039417)]
+traffic_manager.set_path(vehicle, path)
+```
+The `set_route` function allows a custom route to be defined using route commands:
+
+```py
+route = ["Right", "Straight", "Right", "Right", "Left", "Right"]
+ traffic_manager.set_route(vehicle, route)
+```
+
+![set_route](/img/posts/2021-11-07/follow_route.gif){:class="img-fluid rounded mx-auto d-block"}
+
+### Traffic manager controlled vehicle lights
+
+The traffic manager now has functionality to control vehicle lights. Headlights, fog lights and indicators are controlled according to vehicle route data and weather conditions. Indicator lights will activate according to the next planned turn in the route, breaking lights will activate when the vehicle breaks are applied and fog lights and main beams are activated according to the weather and light conditions. During night, the vehicle main beams are activated and in foggy conditions the rear fog lights are activated. 
+
+
+
+## New and Improved Vehicles
+
+### Opening Doors
+
+All generation 1 and generation 2 vehicles now have articulated doors that can be opened and closed through functions in the python API. This functionality is ideal for simulating scenarios in busy city streets and car parks where doors may open unpredictably while driving past or might be blocking access to parking spaces or passageways. 
+
+![doors](/img/posts/2021-11-07/doors.gif){:class="img-fluid rounded mx-auto d-block"}
+
+### Improved vehicle behaviour - gears and breaking
+
+Vehicle physics has been improved with updated breaking distances, acceleration and gear behaviour. Breaking and accelerating behaviour of the vehicles has been carefully remodelled to more accurately capture the behaviour of the CARLA garage vehicles’ real world counterparts, with much care taken to reproduce published breaking distance behaviour as accurately as possible. Similarly the gear change behaviour has been altered to better model real world behaviour gear changes.
```

Comment:
1) It might be just me but I prefer _"updated acceleration, braking and gear behaviours"_. Basically, put say _"acceleration and braking"_, _"braking and acceleration"_ feels weird to me
2) While I very much like the _"has been carefully remodelled to more accurately capture the behaviour of the CARLA garage vehicles’ real world counterparts, with much care taken to reproduce published breaking distance behaviour as accurately as possible"_, I think it might be a very complex sentence that will confuse users.

1) Some misspelling. _brake_, not break. _braking_, not breaking. _remodeled_, with one "l", Either _behaviour_ or _behavior_, but not both

Reply:
changed in new commit